### PR TITLE
GAWB-1790: Entity names in moniter tab now show correctly

### PR DESCRIPTION
**Pull request by ansingh7115**
This fixes a regression in the Monitor tab for submissions with multiple workflows - no matter which workflow was chosen, the breadcrumbs would always show the name of the very first workflow in the list. This was because (duh) we were always choosing the first workflow from the list of workflows when setting the name. I changed it so that the workflow name is chosen based on the workflow id.

Here we're also refactoring the "workflows" object so that it's keywordized

Manually tested to make sure submissions table, workflow details section, and timing diagram are still functional

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs [hunk 1]
```diff
@@ -95,7 +95,7 @@
    :render-workflow-details
    (fn [{:keys [props]} workflowId]
      (let [workflows (:workflows props)
-           workflowName (get-in workflows [0 "workflowEntity" "entityName"])]
+           workflowName (get-in (filterv #(= (get-in % ["workflowId"]) workflowId) workflows) [0 "workflowEntity" "entityName"])]
```
Comment (helgridly) [inline src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs]: my gut says the underlying data structure here wants to be a map of workflow id -> other details, but i don't know the context or structure of the code

Comment (helgridly) [reply src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs]: if a ui grownup is ok with this then i grudgingly am too

Comment (MattPutnam) [reply src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs]: Yeah, this is a little wonky.  But let's not refactor this now, it looks like more of a mess than it's worth.

I would prefer writing this line as:
```clojure
(as-> workflows $
      (filter #(= workflow-id (get % "workflowId")) $)
      (first $)
      (get-in $ "workflowEntity" "entityName"))
```
\<soapbox>If `:workflows` was keywordized, that could be:
```clojure
(->> workflows
     (filter (comp (partial = workflow-id) :workflowId))
     first :workflowEntity :entityName)
```
\</soapbox>

**Comment by ansingh7115** [comment]
Did refactoring to make workflows into a keyword. Ready for re-review

**Comment by MattPutnam** [comment]
👍

**Pull request merged**